### DISCUSSION TITLE
False island detection

### DISCUSSION THREAD

**Discussion by biniecki** [sn4k3/UVtools#379] [Q&A]
I have recently noticed that i get a lot more flags for islands. Upon inspection there usually is support on the layer previous. Its not always in the middle of the island but should still offer support. I'm not sure if the algorithm for this has changed but I have noticed this happening more frequently over the last couple of updates

**Comment by sn4k3** [comment]
No changes was made in Islands since a while...
You need to provide more information and proofs, without it i can't tell you why, but there always an explanation. Most likely settings or unsafe support for the island, renember that 1 px or is not safe to support a large island, i say 1 but it also can be 100px not good to support a 10000px island for example
